src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -548,11 +556,37 @@ public function overrideGlobals()
      *
      * You should only list the reverse proxies that you manage directly.
      *
-     * @param array $proxies A list of trusted proxies
+     * @param array  $proxies       A list of trusted proxies
+     * @param string $trustedHeader Either Request::HEADER_FORWARDED or Request::HEADER_CLIENT_IP to set which header to trust from your proxies
+     *
+     * @throws \InvalidArgumentException When $trustedHeader is invalid
      */
-    public static function setTrustedProxies(array $proxies)
+    public static function setTrustedProxies(array $proxies/*, string $trustedHeader*/)
```

Comment:
it's a reminder, for it to be uncommented in 4.0, as done in such cases usually already